docs/resources/equinix_ecx_l2_connection.md

```diff
@@ -184,8 +184,11 @@ for more details.
 docs.
 
 * `additional_info` - (Optional) one or more additional information key-value objects
-  * `name` - (Required) additional information key
+  * `key` - (Required) additional information key
   * `value` - (Required) additional information value
+
+~> **NOTE:** Connection could be modified from Service Provider Side by using parameters passed to additional_info field:  `{"key": "ASN", "value": "1111"}` `{"key": "Global", "value": "false"}` `{"key": "BGP_IBM_CIDR", "value": "172.16.0.18/30"}` `{"key": "BGP_CER_CIDR", "value": "172.16.0.19/30"}`
```

Comment:
Again, need to make sure this applies to Fabric v3 API. The schema also suggests name/value instead of key/value as well.

Love the initiative to clean up but we need to check it against the correct backend schema :)